src/mds/agg_mds/datastore/elasticsearch_dao.py

```diff
@@ -0,0 +1,234 @@
+from elasticsearch import Elasticsearch
+from typing import List, Dict
+from typing import Any
+import json
+from mds import logger
+
+
+agg_mds_index = "commons-index"
+agg_mds_type = "commons"
+
+
+agg_mds_info_index = "commons-info-index"
+agg_mds_info_type = "commons-info"
+
+
+mapping = {
+    "mappings": {
+        "commons": {
+            "properties": {
+                "__manifest": {
+                    "type": "nested",
+                },
+                "tags": {
+                    "type": "nested",
+                },
+            }
+        }
+    }
+}
+
+elastic_search_client = None
+
+
+async def init(hostname: str = "0.0.0.0", port: int = 9200):
```

Comment:
Should the Kubernetes deployment descriptor's ReadinessProbe be updated to make sure this host:port are responsive before standing up a new pod and adding it to the load balancer rotation?
https://github.com/uc-cdis/cloud-automation/blob/master/kube/services/metadata/metadata-deploy.yaml#L60

Or maybe this check should be included in the logic that is executed as part of the `_status` endpoint (which is currently instrumented by the k8s readinessProbe).
```
@router.get("/_status")
async def get_status():
    now = await db.scalar("SELECT now()")
    return dict(status="OK", timestamp=now)
```

Reply:
You're right. I'll figure out the best way to incorporate that.